src/crashhandler/qgscrashreport.cpp

```diff
@@ -138,3 +150,41 @@ const QString QgsCrashReport::crashID() const
   QString hash = QString( QCryptographicHash::hash( data.toAscii(), QCryptographicHash::Sha1 ).toHex() );
   return hash;
 }
+
+
+void QgsCrashReport::exportToCrashFolder()
+{
+  QString folder = QgsCrashReport::crashReportFolder();
+  QDir dir( folder );
+  if ( !dir.exists() )
+  {
+    QDir().mkdir( folder );
+  }
+
+  QString fileName = folder + "/stack.txt";
+
+  QFile file( fileName );
+  if ( file.open( QIODevice::WriteOnly | QIODevice::Text ) )
+  {
+    QTextStream stream( &file );
+    stream << mStackTrace->fullStack << endl;
+  }
+  file.close();
+
+  fileName = folder + "/report.txt";
+
+  file.setFileName(fileName);
+  if ( file.open( QIODevice::WriteOnly | QIODevice::Text ) )
```

Comment:
Pretty sure you'll need QIODevice::Truncate here too

Reply:
It's a new file each time so should be at the start already.

Reply:
good point!